graphql-query-language/notes/03-query-an-enumeration-type-with-graphql.md

```diff
@@ -33,7 +45,16 @@ If you click the schema tab, you can take a look at this schema.
 
 ## Personal Take
 
-**🔥Tip**: You can hover over one of these field names and press _command_. This will allow you to click on that field, and it'll take you directly to that field definition in the schema.
+**🔥Tip**: You can hover over one of these field names and press _command_. 
+
+<TimeStamp start="1:24" end="1:29">
+
+`⌘` + click on Mac 
+`CTRL` + click on Windows
```

Comment:
I wonder if this note would be more useful if it included the tip context. Users will be able to save notes right?